tests/unit/modules/network/ios/test_ios_vlans.py

```diff
@@ -648,3 +851,52 @@ def test_ios_vlans_gathered(self):
 
         self.maxDiff = None
         self.assertEqual(result["gathered"], gathered)
+
+    def test_ios_vlans_config_gathered(self):
+        set_module_args(dict(configuration=True, state="gathered"))
```

Comment:
This is generating empty config. Please pass a config block

Reply:
Thanks for the review, Sagar. Fixed.